src/xdatasets/utils.py

```diff
@@ -105,7 +105,7 @@ def cache_catalog(url):
     opener = urllib.request.build_opener(proxy)
     urllib.request.install_opener(opener)
 
-    tmp_dir = Path(tempfile.gettempdir()).joinpath("catalogs")
+    tmp_dir = Path(tempfile.mkdtemp()).joinpath("catalogs")
```

Comment:
Maybe this is actually better, if we want it to act as a cache?
```suggestion
    tmp_dir = Path.home() / ".cache" / "xdatasets" / "catalogs"
```

Reply:
We could create a name based on a hashed date string if we only want it around temporarily (<24 hours)?

Reply:
but to be fair, it was not really a cache for long before as /tmp still gets erased evry few days/weeks ?

Reply:
do you guys like:
`Path(tempfile.gettempdir()).joinpath(getpass.getuser()).joinpath("catalogs")` ?

Reply:
Sounds good!

Reply:
Yeah, that could work for now.